airbyte-config/models/src/main/resources/types/AttemptFailureSummary.yaml

```diff
@@ -0,0 +1,24 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
+"$id": https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/AttemptFailureSummary.yaml
+title: AttemptFailureSummary
+description: Attempt-level summarization of failures that occurred during a sync workflow.
+type: object
+additionalProperties: false
+required:
+  - failures
+properties:
+  failures:
+    description: Ordered list of failures that occurred during the attempt.
+    type: array
+    items:
+      "$ref": FailureReason.yaml
+  partialSuccess:
+    description: True if the number of committed records for this attempt was greater than 0. False if 0 records were committed.
+    type: boolean
+  cancelled:
+    description: Indicates whether this attempt was manually cancelled.
+    type: boolean
+  cancelledBy:
+    description: The user who cancelled the attempt.
+    type: string # TODO how do we represent users?
```

Comment:
it should refer to whatever uuid we use in the user table in the cloud database i think.